### ISSUE TITLE
The scanning number is higher than the migrated data

### ISSUE THREAD

**Issue by giulianisanches** [redis-developer/riot#110]
Hi everyone,

When I run the following command, note the amount of scanned items is 3069 (witout the `--scan-match` is 56000).

```shell
[ec2-user@host ~]$ riot-redis --stacktrace -c -h "$ORIGIN_REDIS_ENDPOINT" -p "$REDIS_PORT" -a "$REDIS_PASSWORD" --tls replicate-ds -c -h "$DESTINATION_REDIS_ENDPOINT" -p "$REDIS_PORT" -a "$REDIS_PASSWORD" --tls --scan-match='l_*' --reader-batch=50 --batch 25 --mode snapshot --no-verify
Scanning 100% ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3069/3069 (0:00:03 / 0:00:00) 1023.0/s
```

But when i log to both origin and target redis using the `redis-cli` and run a `keys *` command total item count is 600 in both sides.

What does the scanning information means ?

**Comment by jruaux** [comment]
It looks like the scan size estimate was not accurate. It will be fixed in the upcoming release of RIOT

**Comment by jruaux** [comment]
🎉 This issue has been resolved in `v3.1.1` ([Release Notes](https://github.com/redis-developer/riot/releases/tag/v3.1.1))
